templates/data/spark/support.html

```diff
@@ -0,0 +1,463 @@
+                </div>
+                <hr class="p-rule--muted" />
+                <div class="row p-section--shallow">
+                    <div class="col-3 col-medium-3">
+                        <h3 class="p-heading--5">Spark data lake platform deployment service</h3>
+                    </div>
+                    <div class="col-6 col-medium-3">
+                        <p>
+                            Canonical engineers deploy Charmed Spark to the cloud or data centre of your choice. Kubernetes and air-gapped environments are fully supported.
+                        </p>
+                        <hr class="p-rule--muted" />
+                        <p>
+                            <a href="https://ubuntu.com/pricing/consulting">Explore pricing&nbsp;&rsaquo;<span class="u-off-screen">explore pricing for consultancy</span></a>
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="p-section">
+        <div class="row">
+            <hr class="p-rule" />
+            <div class="col-3 col-medium-6">
+                <h2>Training</h2>
+            </div>
+            <div class="col-3 col-medium-3">
+                <h3 class="p-heading--5">Charmed Spark training</h3>
+            </div>
+            <div class="col-6 col-medium-3">
+                <p>
+                    Get hands-on training with Spark and Charmed Spark, Canonical's solution for Apache Spark, in a three day workshop delivered remotely or on-site.
+                </p>
+                <hr class="p-rule--muted" />
+                <p>
+                    <a class="p-button" href="/contact-us" aria-controls="data-spark-modal">Get in touch<span class="u-off-screen">Get in touch</span></a>
+                </p>
+            </div>
+        </div>
+    </section>
+
+    <hr class="p-rule is-fixed-width" />
+
+    <section class="p-strip is-deep">
+        <div class="u-fixed-width">
+            <p class="p-heading--2">
+                <a href="/contact-us" aria-controls="data-spark-modal">Contact us to get support for Spark&nbsp;&rsaquo;</a>
+            </p>
+        </div>
+    </section>
+
+    <section class="p-section">
+        <div class="row--50-50">
+            <hr class="p-rule" />
+            <div class="col p-section">
+                <h2>
+                    Come for Spark support,
+                    <br class="u-hide-small" />
+                    get the full stack
+                </h2>
+            </div>
+            <div class="col p-section">
+                <p>
+                    With Ubuntu Pro + Support for Apache Spark, you get an all-access pass to everything Canonical has to offer, including best-in-class open source solutions for enterprise data management and AI. Ubuntu Pro subscriptions are competitively priced per server, not per application.
+                </p>
+            </div>
+        </div>
+        <div class="row">
```

Comment:
On the Figma design, it shows some spacing between the logo rows
<img width="851" alt="Screenshot 2024-10-24 at 2 27 19 PM" src="https://github.com/user-attachments/assets/ce489508-1f2d-4849-9445-06d74a4f90b6">